packages/runtime/src/runtime/menu.tsx

```diff
@@ -36,27 +46,32 @@ interface MenuItem {
   divider?: boolean;
   hasNotifications?: boolean;
 }
-
-interface MenuBaseProps {
-  items: MenuItem[];
-  styles?: {
+interface MenuStyles {
+  backgroundColor?: TypeColors;
+  labelColor?: TypeColors;
+  selectedColor?: TypeColors;
+  labelFontSize?: number;
+  searchBoxColor?: TypeColors;
+  iconWidth?: string;
+  iconHeight?: string;
+  width?: string;
+  headerCollapsedWidth?: string;
+  footerCollapsedWidth?: string;
+  onSelectStyles?: {
     backgroundColor?: TypeColors;
-    labelColor?: TypeColors;
-    selectedColor?: TypeColors;
-    labelFontSize?: number;
-    searchBoxColor?: TypeColors;
-    iconWidth?: string;
-    iconHeight?: string;
-    width?: string;
-    onSelectStyles?: {
-      backgroundColor?: TypeColors;
-      borderRadius?: string;
-    };
+    borderRadius?: string;
```

Comment:
Let's clean this up a bit better - consolidate the selected styles in `onSelectStyles` and remove irrelevant styles like `searchBoxColor`.